packages/near-fast-auth-signer/src/components/SignMultichain/SignMultichain.tsx

```diff
@@ -19,7 +20,11 @@ import { getAuthState } from '../../hooks/useAuthState';
 import useIframeDialogConfig from '../../hooks/useIframeDialogConfig';
 import InternetSvg from '../../Images/Internet';
 import ModalIconSvg from '../../Images/ModalIcon';
+import WarningCircle from '../../Images/WarningCircle';
```

Comment:
I think we have a inconsistent naming here. As you can see the `InternetSvg` and `ModalIconSvg`are named with SVG at the end.